website/sidebars.json

```diff
@@ -1,5 +1,46 @@
 {
   "docs": {
-    "General": ["meetings", "resources", "about"]
+    "General": [
+      "meetings",
+      "resources",
+      "about"
+    ],
+    "OSR SIG Materials": [
+      {
+        "type": "subcategory",
+        "label": "Goals and Objectives",
+        "ids": [
+          "goals-objectives/resources"
+        ]
+      },
+      {
+        "type": "subcategory",
+        "label": "Corportate Alignment",
```

Comment:
Typo, should be 'Corporate'

Reply:
Hi @psmulovics - The fix has been contributed as PR #30 that needs to be reviewed and merged 👍🏻